cypress/e2e/app_tests/testsHW12/step-by-stepTextChecking.cy.js

```diff
@@ -0,0 +1,28 @@
+///<reference types="cypress" />
+
+//const { Button } = require("bootstrap");
+
+describe("tests HW12 suite", () => {
+  it("step-by-stepTextChecking", () => {
+    cy.visit("pages/layout/stepper");
+
+    cy.get(".lorem.ng-star-inserted").contains("Proin varius");
+    cy.get(
+      ".appearance-filled.size-medium.status-primary.shape-rectangle.ng-star-inserted.transitions",
+    )
+      .eq(3)
+      .click();
+    cy.get(".lorem.ng-star-inserted").contains("Curabitur luctus");
+    cy.get(
+      ".appearance-filled.size-medium.status-primary.shape-rectangle.ng-star-inserted.transitions",
+    )
+      .eq(3)
+      .click();
+    cy.get(".lorem.ng-star-inserted").contains("Proin varius");
+    cy.get(
+      ".appearance-filled.size-medium.status-primary.shape-rectangle.ng-star-inserted.transitions",
+    )
+      .eq(3)
+      .click();
```

Comment:
you can use alias for your code
```suggestion
 cy.get(".lorem.ng-star-inserted").as("headerText");
cy.get('.vertical ').contains('next').as("nextButton")

 cy.get("@headerText").should("contain","Proin varius" )
cy.get("nextButton").click()

 cy.get("@headerText").should("contain","Curabitur luctus");
cy.get("nextButton").click()

cy.get("@headerText").should("contain","Proin varius" )
cy.get("nextButton").should("be.disabled")
```